lte/gateway/c/oai/tasks/ngap/ngap_amf_handlers.c

```diff
@@ -841,6 +854,354 @@ int ngap_handle_new_association(
   OAILOG_FUNC_RETURN(LOG_NGAP, RETURNok);
 }
 
+int ngap_amf_handle_ue_context_release_request(
+    ngap_state_t* state, __attribute__((unused)) const sctp_assoc_id_t assoc_id,
```

Comment:
linter is marking `state` as unused here, probably because the only usage is commented out, it's fine if later it's gonna be used but if not, this param can be removed